tests/headers/float128.hpp

```diff
@@ -0,0 +1,7 @@
+// FIXME: We have no way to get 128 bit aligned structs in Rust at the moment,
+// and therefore the generated layout tests for this struct will fail. When we
+// can enforce 128 bit alignment, we can re-enable this test.
+//
+// struct A {
+//     __float128 f;
+// };
```

Comment:
Would it make sense to have a global constant __float128 gloablaFloat to test it a bit.
This does not generate layout tests, I think.